lib/player.js

```diff
@@ -2278,7 +2278,8 @@ shaka.Player = class extends shaka.util.FakeEventTarget {
                         'arbitrary language initially');
     }
 
-    if (this.isLive() && this.config_.streaming.liveSync) {
+    if (this.isLive() && (this.config_.streaming.liveSync ||
+        this.manifest_.serviceDescription)) {
```

Comment:
In the parser you are always defining a value for this.manifest_.serviceDescription, so this condition is no longer valid, in the parsers the default value must be null

Reply:
I have changed the parser to make serviceDescription optional, because it is optional in the MPD